src/Api/Dfe.Complete.Api/Controllers/TasksDataController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Asp.Versioning;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using Dfe.Complete.Application.Projects.Models;
+using Microsoft.AspNetCore.Authorization;
+using Dfe.Complete.Application.Projects.Queries.GetTransferTasksData;
+using Dfe.Complete.Application.Projects.Commands.UpdateProject;
+using Dfe.Complete.Application.Projects.Queries.TaskData;
+
+namespace Dfe.Complete.Api.Controllers
+{
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Authorize(Policy = "CanRead")]
+    [Route("v{version:apiVersion}/[controller]")]
+    public class TasksDataController(ISender sender) : ControllerBase
+    {
+        /// <summary>
+        /// Gets the transfer tasks data by Id.
+        /// </summary>
+        /// <param name="request">The transfer tasks data Id.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [Authorize(Policy = "CanRead")]
+        [HttpGet]
+        [Route("TaskData/Transfer")]
+        [SwaggerResponse(200, "Transfer tasks data returned successfully.", typeof(TransferTaskDataDto))]
+        [SwaggerResponse(404, "Transfer tasks data not found for the given Id.")]
+        public async Task<IActionResult> GetTransferTasksDataByIdAsync([FromQuery] GetTransferTasksDataByIdQuery request, CancellationToken cancellationToken)
+        {
+            var transferTasksData = await sender.Send(request, cancellationToken);
+
+            return Ok(transferTasksData.Value);
+        }
+        /// <summary>
+        /// Updates the handover with delivery officer for either conversion or transfer tasks data.
+        /// </summary>
+        /// <param name="request">The update task data command.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [Authorize(Policy = "CanReadWriteUpdate")]
+        [HttpPatch]
+        [Route("TaskData/Handover/DeliverOfficer")]
```

Comment:
I suspect this should be delivery officer
```suggestion
        [Route("TaskData/Handover/DeliveryOfficer")]
```

Reply:
Resolved now